### ISSUE TITLE
Compliance Presentation Exchange v2.x.x

### ISSUE THREAD

**Issue by n0900** [a-sit-plus/vck#110]
Need to update our references from presentation exchange 1.0.0 to 2.0.0 or later.
Also necessitates potentially updating relevant classes.
Notably `InputDescriptor` does no longer contain member `schema`

**Comment by JesusMcCloud** [comment]
still relevant @n0900 ?

**Comment by n0900** [comment]
Yes, changed the mentioned `InputDescriptor` but have not touched any other part of the spec
